src/components/security/SecurityScansTab.tsx

```diff
@@ -207,10 +211,7 @@ export class SecurityScansTab extends Component<RouteComponentProps<{}>, Securit
   }
 
   changePageSize(newPageSize: number): void {
```

Comment:
Search is cleared when page size changes, why ?
we should retain the  search even if user change the page size